java-library/src/main/java/com/microsoft/azure/functions/sql/annotation/SQLInput.java

```diff
@@ -25,12 +25,12 @@
     /**
      * Query string or name of stored procedure to be run.
      */
-    String commandText() default "";
+    String commandText();
 
     /**
      * Text or Stored Procedure.
      */
-    String commandType() default "";
+    String commandType();
```

Comment:
This we should default to Text if possible - that's the default we do for the other langs

Reply:
I tried removing commandType from non-CSharp samples that include parameters and the parameters were not getting set correctly. When I debugged the [GetProducts](https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/samples-js/GetProducts/function.json#L23) function, it hits this constructor https://github.com/Azure/azure-functions-sql-extension/blob/f01ffa47e887baaf5cd33cea8103f2c8c928a0b1/src/SqlAttribute.cs#L39 and sets the parameter to null. 

I think we need to add another constructor that includes parameters. I will send a PR out for that.

Right now our docs and samples still note that commandType is required in non-CSharp languages. https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-input?tabs=in-process&pivots=programming-language-javascript#configuration

Reply:
Gotcha, yeah we can make that improvement. But you should still be able to give this a default value here, right? It'll end up having a value - it's just one the user doesn't necessarily need to specify.

Reply:
For some reason when I put a default value (CommandType.Text), the GetProducts function still uses the SqlAttribute constructor that sets the parameters to null. Once I add a new constructor that sets parameters, I'll update the java library to include a default value for CommandType.